### PR TITLE
Remove personal data from mod_offline

### PR DESCRIPTION
**Pull request by mkacper**
This PR addresses **MIM-336**.

### CODE REVIEW

File: big_tests/tests/gdpr_SUITE.erl [hunk 1]
```diff
@@ -196,16 +202,18 @@ pick_backend_for_mam() ->
 mam_required_modules(Backend) ->
     [{mod_mam_meta, [{backend, Backend}, {pm, []}]}].
 
-pick_backend_for_vcard() ->
+pick_backend_for_vcard_or_offline() ->
```
Comment (ludwikbukowski) [inline big_tests/tests/gdpr_SUITE.erl]: Can we rename it as `pick_backend_as_riak_or_rdbms` ? it fits better what it does (there is nothing specific aboud vcard or offline in the function itself).

**Comment by mongoose-im** [comment]
[6521.1](https://travis-ci.org/esl/MongooseIM/jobs/534754062) / Erlang 19.3 / small_tests / a0a3052fd4f82344e7bfc1558a97b3bff8b7759b
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2312/6521/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2312/6521/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-20_11.17.51/lib.mongooseim.logs/run.2019-05-20_11.17.51/suite.log.html)

**Comment by mongoose-im** [comment]
[6534.1](https://travis-ci.org/esl/MongooseIM/jobs/535195350) / Erlang 19.3 / small_tests / 11f266668c1fc8939b6e7560c4bd03b9e55964b8
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2312/6534/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2312/6534/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-21_09.02.42/lib.mongooseim.logs/run.2019-05-21_09.02.43/suite.log.html)

**Comment by mongoose-im** [comment]
[6540.1](https://travis-ci.org/esl/MongooseIM/jobs/535230559) / Erlang 19.3 / small_tests / 3ae62af6b7d63b34c5d20999e00860f8b72e9ce9
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2312/6540/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2312/6540/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-21_14.24.56/lib.mongooseim.logs/run.2019-05-21_14.24.57/suite.log.html)

**Pull request merged**